package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "1.0.57",
+  "version": "1.0.58",
```

Comment:
cc @constanzauanini there's a conflict :) https://github.com/terraware/web-components/pull/124

Reply:
Ups sorry, yes, I already merge. Could you change it to 59?